src/regexp.c

```diff
@@ -848,7 +848,6 @@ Reprog *re_regcomp(const char *pattern, int cflags, const char **errorp)
         g->prog = prog;
 	g->pstart = g->pend = rd_malloc(sizeof (Renode) * strlen(pattern) * 2);
 
-        memcpy(((char *)prog)+1, prog, 4);
```

Comment:
i assume you know what you're doing.

Reply:
More importantly I have no idea what I was doing adding that line!